html/pages/about.inc.php

```diff
@@ -237,5 +237,5 @@
 
     // convert the version date from ISO 8601 to the browser's timezone
```

Comment:
Done.

Wouldn't have mattered using moment, they fall back to new Date anyway.